src/explore-education-statistics-common/src/services/publicationService.ts

```diff
@@ -91,18 +91,22 @@ export interface ContentSection<BlockType> {
   content: BlockType[];
 }
 
-export const publicationSortOptions = [
-  'newest',
-  'oldest',
-  'alphabetical',
-] as const;
+export const publicationSortOptions = ['newest', 'oldest', 'title'] as const;
 
 export type PublicationSortOption = typeof publicationSortOptions[number];
 
-// TODO EES-3517 expand to include filters and search
-export interface ListPublicationsRequest {
+export type PublicationSortParam = 'published' | 'title' | 'relevance';
+
+export type PublicationOrderParam = 'asc' | 'desc';
+
+interface GetPublicationsRequest {
```

Comment:
Would suggest naming this `PublicationListRequest`.

Whilst it's not a big big deal, would say it's a good idea for type names to follow similar naming conventions as component file names.

This means nouns should go from less to more specific terminology i.e. `Publication` is the least specific thing, whilst the `ListRequest` is the most specific thing hence `PublicationListRequest` instead of `ListPublicationRequest`.

We're already doing this with `PublicationSortParam` and `PublicationOrderParam` so we should be consistent.